dockers/docker-orchagent/enable_counters.py

```diff
@@ -17,6 +17,7 @@ def enable_counters():
     enable_counter_group(db, 'PFCWD')
     enable_counter_group(db, 'PG_WATERMARK')
     enable_counter_group(db, 'QUEUE_WATERMARK')
+    enable_counter_group(db, 'BUFFER_POOL')
```

Comment:
Correct counter group name is BUFFER_POOL_WATERMARK

```suggestion
    enable_counter_group(db, 'BUFFER_POOL_WATERMARK')
```